src/main/scala/is/hail/methods/LDMatrix.scala

```diff
@@ -18,10 +18,11 @@ object LDMatrix {
     */
   def apply(vds : VariantDataset, optComputeLocally: Option[Boolean]): LDMatrix = {
     val nSamples = vds.nSamples
-    val originalVariants = vds.variants.collect()
+    val persistedVDS = vds.persist()
```

Comment:
I added an unpersist, but this whole thing is unnecessary now that I made LD Matrix less shitty with my other PR. Once that goes in, I should just rebase on that and get rid of this entirely.